src/datachain/lib/clip.py

```diff
@@ -130,17 +130,20 @@ def clip_similarity_scores(
         ```
     """
 
+    device = next(model.parameters()).device
```

Comment:
Probably worth noting for future reference that this is not guaranteed to work but is likely good enough for almost all use cases. See https://stackoverflow.com/questions/58926054/how-to-get-the-device-type-of-a-pytorch-module-conveniently.

Reply:
Maybe the function should also include a `device` parameter but default to this if no device is passed?

Reply:
I suggest this first. Let's add `device`.